scripts/authenticator_regenerate.sh

```diff
@@ -37,19 +37,6 @@ cd $repodir
   cargo run -p aptos-openapi-spec-generator -- -f json -o api/doc/spec.json
 )
 
-(
-  echo
-  echo "Regenerating Typescript SDK (in `pwd`)"
-  # Typescript SDK client files
-  cd ecosystem/typescript/sdk
-  pnpm install
-  pnpm generate-client
-
-  # Typescript SDK docs
-  pnpm generate-ts-docs
-  cd ..
-)
```

Comment:
Why is this removed btw?  Though, I guess the SDK doesn't need to be built anyways, it'll be deprecated.

Reply:
`pnpm generate-ts-docs` gives the following error, and Maayan also told me that we don't need to update it.
```
Cloning into '/tmp/ts-sdk-doc'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```